README.md

```diff
@@ -8,10 +8,24 @@ Resume.
 The code in this repository is experimental and is provided without any warranty. However, it should be relatively easy
 to adapt for your own use.
 
+
+## Building and running
+To build from source and run in dev mode:
+
+```
+go run ./cmd/vmproxy <vm name> <vnc addrport>
+```
+
+For the initial run you need to register with `TS_AUTHKEY`.
+
+Note: requires Go 1.20
```

Comment:
```suggestion
Note: Requires Go 1.20
```

Reply:
oh darn... missed that 🤦

Reply:
Punctuation rules as I know didn't capitalize unless it was followed by a full sentence. Here is more of a statement, 'that is to say'.